Scripts/utils.py

```diff
@@ -145,25 +147,38 @@ def expand_ranges(stringInput):
 
 
 ############################### User Choice #################################
-# User inputs Y/N for choice, returns True or False
-# Takes in message to display
 
-def choice(message="", bypass=False):
+def choice( message: str='', bypass: bool=False, addCancel: bool=False ) -> Union[bool, None]:
+  '''User inputs 'yes,no' for choice.
+    If argument addCancel is True the 'cancel' option appear
+  '''
+
   if bypass == True:
     return True
 
-  # While loop until valid input
-  valid = False
-  while valid == False:
-    response = input("\n" + message + f" ({F.LIGHTCYAN_EX}y{S.R}/{F.LIGHTRED_EX}n{S.R}): ").strip()
-    if response == "Y" or response == "y":
-      return True
-    elif response == "N" or response == "n":
-      return False
-    elif response == "X" or response == "x":
-      return None
-    else:
-      print("\nInvalid Input. Enter Y or N  --  Or enter X to return to main menu.")  
+  choices = [
```

Comment:
maybe it can be simpler like below

```python
def choice( message: str='', bypass: bool=False):
  # ...
  choices_dict = {
      'y': ['yes', True],
      'n': ['no', False],
      'x': ['cancel', None],
  }
  question = [
    inquirer.List('choice',
      message=message,
      choices=[(val[0], key) for key, val in choices_dict.items()],
    ),
  ]
  return choices_dict[inquirer.prompt(question)['choice']][1]
```

i also found out that `inquirer.prompt(question)` return `dict[Any, Any] | None` so that None value should be handle or raise error when it happen